pkg/skaffold/build/gcb/cloud_build.go

```diff
@@ -195,8 +194,7 @@ watch:
 			if err == nil {
 				return true, nil
 			}
-			if strings.Contains(err.Error(), "Error 429: Quota exceeded for quota metric 'cloudbuild.googleapis.com/get_requests'") {
-				// if we hit the rate limit, continue to retry
```

Comment:
I'd say we should leave this comment here or leave a new comment explaining this case since it's not really clear what this is to someone reading the code

Reply:
Hi, I have returned the comments and added another one on 429.